### PR TITLE
Enable HTTP method Override

### PR DESCRIPTION
**Pull request by praneethk2094**

### CODE REVIEW

File: content/millennium/r4.md [hunk 1]
```diff
@@ -414,6 +414,17 @@ Where possible, the FHIR<sup>®</sup> standard strives to use appropriate HTTP v
  `POST` | Used for creating resources.
  `PUT`  | Used for updating resources.
 
+### HTTP method Override
+If your firewall rules do not support HTTP methods like PUT, PATCH or DELETE, the request can be made through POST using one of the following:
```
Comment (daliboz) [inline content/millennium/r4.md]: This wouldn't be used for firewall issues. This would be used if the client library doesn't support those methods.

Comment (praneethk2094) [reply content/millennium/r4.md]: How about i make it more generic? something like
 _If support fo HTTP methods like PUT, PATCH or DELETE is unavailable, the request can be.._

Comment (praneethk2094) [reply content/millennium/r4.md]: https://github.com/cerner/fhir.cerner.com/pull/263/commits/76e9210f64aeffa0baa8b15143892df2251195eb

File: content/millennium/r4.md [hunk 2]
```diff
@@ -414,6 +414,17 @@ Where possible, the FHIR<sup>®</sup> standard strives to use appropriate HTTP v
  `POST` | Used for creating resources.
  `PUT`  | Used for updating resources.
 
+### HTTP method Override
```
[the remark this answers is not in the document]
Comment (praneethk2094) [reply content/millennium/r4.md]: https://github.com/cerner/fhir.cerner.com/pull/263/commits/82dfd84ed0bbc2dd1eae2c9e283c7407817810e1#diff-927cb2929bf7276c241f24d25c9c2dddR417

File: content/millennium/r4.md [hunk 3]
```diff
@@ -414,6 +414,17 @@ Where possible, the FHIR<sup>®</sup> standard strives to use appropriate HTTP v
...
+If there is no support for HTTP methods such as PUT, PATCH or DELETE, the request can be made through POST using one of the following:
```
[the remark this answers is not in the document]
Comment (praneethk2094) [reply content/millennium/r4.md]: https://github.com/cerner/fhir.cerner.com/pull/263/commits/82dfd84ed0bbc2dd1eae2c9e283c7407817810e1#diff-927cb2929bf7276c241f24d25c9c2dddR417

**Pull request merged**